BrightCoders-Proyectos/tribo-mobile
# Feature/4 setting screens text

# Description
This PR is linked to issue #4 
- I created a folder which contains a file to storage in a variable all the screens text in order to implement it properly in the screens UI.

 # Testing
Steps to verify it: 
1. Go to project folder (tribo-mobile\tribomobile).
2.  npm install.
3. npx react-native start.
4. npx react-native run-android

if you want to test another string follow next steps:
1. Create a ![image](https://user-images.githubusercontent.com/69820530/101667194-a52ed980-3a14-11eb-82f2-bd4362bf1a7d.png) tag.
3. Call the import and any property of the contentText file like this one ![image](https://user-images.githubusercontent.com/69820530/101669061-0788d980-3a17-11eb-9936-9f02d6e81eac.png)


 
 Example:
  - I have created a text tag to implement it and show it in App.js
  
  ## Screenshots 
![image](https://user-images.githubusercontent.com/69820530/101666170-7fed9b80-3a13-11eb-976c-e1a7886c4cb1.png)
